app/assets/stylesheets/modules/projects.css.scss

```diff
@@ -0,0 +1,72 @@
+.project {
+
+  min-height: calc(100vh - 67px);
+  display: flex;
+
+  .project-info-sidebar {
+    color: #fff;
+    width: 300px;
+    padding: 0 20px;
+    box-shadow: 5px 0 0 rgba(0,0,0,0.1);
+    z-index: 1;
+  }
+
+  .project-collections {
+    background: #e3e3e3;
```

Comment:
@jsiarto, I think we should use color variables for most (all?) of these colors. Let's leverage variables and `darken`/`lighten` as much as possible and where appropriate.